frontend/src/app/sidebar/sidebar.component.html

```diff
@@ -1,7 +1,124 @@
-&nbsp;
-<ul id="sidebar-nav" class="usa-sidenav-list" [style.bottom]="bottom" [style.top]="top" [style.position]="position">
-  <li class="usa-current" *ngFor="let item of items">
-    <a [ngClass]="{'usa-current': util.currentSection && util.currentSection === util.createId(item.sectionName) }"
-      href="#" (click)="gotoHashtag(util.createId(item.sectionName), $event)">{{item.sectionName}}</a>
-  </li>
-</ul>
+<ng-container *ngIf="!mobileMenu">
+  &nbsp;
+  <ul id="sidebar-nav" class="usa-sidenav-list" [style.bottom]="bottom" [style.top]="top" [style.position]="position">
+    <li class="usa-current" *ngFor="let item of items">
+      <a [ngClass]="{'usa-current': util.currentSection && util.currentSection === util.createId(item.sectionName) }"
+        href="#" (click)="util.gotoHashtag(util.createId(item.sectionName), $event)">{{item.sectionName}}</a>
+    </li>
+  </ul>
+</ng-container>
+
+<ng-container *ngIf="mobileMenu">
+  <div class="usa-overlay" [ngClass]="{'is-visible' : showMobileNav}"></div>
+  <button id="mobile-menu-btn" class="usa-menu-btn mobile-menu-btn" (click)='toggleMobileNav()' [style.position]="menuBtnPosition" [style.top]="menuBtnTop">Menu</button>
+  <nav role="navigation" [ngClass]="{'is-visible' : showMobileNav}" class="usa-nav sidenav-mobile">
+    <button class="usa-nav-close" (click)="toggleMobileNav()">
+      <img src="/assets/img/close.svg" alt="close">
+    </button>
+    <ul id="sidenav-mobile" class="usa-sidenav-list usa-accordion">
+
+      <li *ngFor="let item of items">
+        <a href="#" id="mobile-{{item.id}}-link" (click)="util.gotoHashtag(item.id, $event); toggleMobileNav();">{{item.title}}</a>
+        <ul class="usa-sidenav-sub_list" *ngIf="item.sections">
+          <li *ngFor="let section of item.sections">
+            <a href="#" id="mobile-{{section.id}}-link" (click)="util.gotoHashtag(section.id, $event); toggleMobileNav();">{{section.title}}</a>
+          </li>
+        </ul>
+      </li>
+
+<!--
```

Comment:
Please remove this commented out commented out code.

Reply:
Task is added to remove that.